ArduCopter/Attitude.cpp

```diff
@@ -188,6 +188,25 @@ float Copter::get_surface_tracking_climb_rate(int16_t target_rate)
 #endif
 }
 
+// get surfacing tracking alt
+// returns true if there is a valid target
+bool Copter::get_surface_tracking_target_alt_cm(float &target_alt_cm) const
```

Comment:
These should probably be `Copter::Mode::SurfaceTracking.set_alt_target_cm(...)` etc